### PR TITLE
Rename template seed tasks to fit in with the rest of the stock tasks

### PR DESCRIPTION
**Pull request by robacarp**
The rest of the tasks in `lucky -h` are formatted using [reverse domain name notation](https://en.wikipedia.org/wiki/Reverse_domain_name_notation):

db.schema.dump / db.schema.restore
db.migrate / db.migrate.one
gen.action.api / gen.action.browser

This changes the default generated tasks:

- `db.create_required_seeds` to `db.seed.required_data`
- `db.create_sample_seeds` to `db.seed.sample_data`

To match the existing notation.

```diff
lucky -h
Usage: lucky name.of.task [options]

Available tasks:

  ▸ build.release          Compile app for production
  ▸ db.create              Create the database
-  ▸ db.create_required_seeds  Add database records required for the app to work
-  ▸ db.create_sample_seeds    Add sample database records helpful for development
  ▸ db.drop                Drop the database
  ▸ db.migrate             Run any pending migrations
  ▸ db.migrate.one         Run just the next pending migration
  ▸ db.migrations.status   Print the current status of migrations
  ▸ db.redo                Rollback and run just the last migration
  ▸ db.reset               Drop, recreate, and run migrations.
  ▸ db.rollback            Rollback the last migration
  ▸ db.rollback_all        Rollback all migrations
  ▸ db.rollback_to         Rollback to a specific migration
  ▸ db.schema.dump         Export database schema to a sql file
  ▸ db.schema.restore      Restore database from a sql dump file
+  ▸ db.seed.required_data  Add database records required for the app to work
+  ▸ db.seed.sample_data    Add sample database records helpful for development
  ▸ db.setup               Runs a few tasks for setting up your database
  ▸ db.verify_connection   Verify connection to postgres
  ▸ exec                   Execute code. Use this in place of a console/REPL
  ▸ gen.action.api         Generate a new api action
  ▸ gen.action.browser     Generate a new browser action
  ▸ gen.component          Generate a new HTML component
  ▸ gen.migration          Generate a new migration
  ▸ gen.model              Generate a model, query, and save operation
  ▸ gen.page               Generate a new HTML page
  ▸ gen.resource.browser   Generate a resource (model, operation, query, actions, and pages)
  ▸ gen.secret_key         Generate a new secret key
  ▸ routes                 Show all the routes for the app
  ▸ watch                  Start and recompile project when files change
```

Still to do:

- [x] decide on a filename for these two files

### CODE REVIEW

File: src/web_app_skeleton/tasks/db_seed_required_data.cr
Comment (robacarp) [inline src/web_app_skeleton/tasks/db_seed_required_data.cr:9]: I renamed this file but I'm not sure about the new name. I considered `db/seeds/required_data.cr` but that felt like a big enough change that it warranted discussion.

What is a good naming convention for these files?

Comment (matthewmcgarvey) [reply src/web_app_skeleton/tasks/db_seed_required_data.cr:9]: I'm fine with this class name, but could you update the file structure to match? `tasks/db/seed/required_data.cr`

Comment (robacarp) [reply src/web_app_skeleton/tasks/db_seed_required_data.cr]: All set

**Review by matthewmcgarvey — APPROVED** [review]
I'm fine with this, but we might wait for a second approval

**Review by stephendolan — APPROVED** [review]
This is a great suggestion! Thanks for submitting.

**Pull request merged**
